docs/technical-reference.md

```diff
@@ -287,24 +287,40 @@ and hybrid flows, and check the box to issue ID tokens
 
 ### Okta
 
-1. Go to Okta > Admin > Applications
-1. Click `Add Application`
-1. Select type `Web` 
-1. Name the app `Quilt` or something similar
-1. Configure the app as shown below
-1. Add `<QuiltWebHost>` to `Login redirect URIs` and
-`Initiate login URI`
-1. Copy the `Client ID` to a safe place
-1. Go to API > Authorization servers
-1. You should see a default URI that looks something like the following:
+Note: You will need Okta administrator privileges to add a new Application.
+
+1. Go to **Okta > Admin > Applications > Applications**
+
+![](./imgs/okta-add-application.png)
+
+2. Click `Create App Integration`. A new modal window opens.
+3. Assign `Sign-in method` radio button to `OIDC - OpenID Connect`.
+4. Assign `Application type` radio button to `Web Application`.
+
+![](./imgs/okta-add-application-modal.png)
```

Comment:
@robnewman be sure to "crush" all images with ImageOptim or PNG crush or similar so repo doesn't bloat.

Reply:
All net-new images added are optimized